ci/benchmark/build.sh

```diff
@@ -82,8 +85,8 @@ if [[ "${INSTALL_DASK_MAIN}" == 1 ]]; then
     gpuci_logger "gpuci_mamba_retry update dask"
     gpuci_mamba_retry update dask
 else
-    gpuci_logger "gpuci_mamba_retry install conda-forge::dask>=2022.7.1 conda-forge::distributed>=2022.7.1 conda-forge::dask-core>=2022.7.1 --force-reinstall"
-    gpuci_mamba_retry install conda-forge::dask>=2022.7.1 conda-forge::distributed>=2022.7.1 conda-forge::dask-core>=2022.7.1 --force-reinstall
+    gpuci_logger "gpuci_mamba_retry install conda-forge::dask=={$DASK_STABLE_VERSION} conda-forge::distributed=={$DASK_STABLE_VERSION} conda-forge::dask-core=={$DASK_STABLE_VERSION} --force-reinstall"
+    gpuci_mamba_retry install conda-forge::dask=={$DASK_STABLE_VERSION} conda-forge::distributed=={$DASK_STABLE_VERSION} conda-forge::dask-core=={$DASK_STABLE_VERSION} --force-reinstall
```

Comment:
Could we try out

```suggestion
    gpuci_logger "gpuci_mamba_retry install dask=={$DASK_STABLE_VERSION}"
    gpuci_mamba_retry install dask=={$DASK_STABLE_VERSION}
```

The reasoning behind this is:

- the `==` should force an exact match on `DASK_STABLE_VERSION`, which isn't contained in the nightly channel and would force us to install stable `dask` from conda-forge
- the updates to the `dask`/`distributed` feedstocks should make it so this will always pull in stable `distributed`/`dask-core` versions

If this method doesn't work we can always stick with what you have hear, just want to confirm if the fixes actually succeeded 🙂